crates/ruff_linter/src/rules/flake8_import_conventions/rules/unconventional_import_alias.rs

```diff
@@ -74,6 +74,10 @@ pub(crate) fn unconventional_import_alias(
         return;
     }
 
+    if expected_alias == "__debug__" {
+        return;
+    }
+
```

Comment:
Is this check still required, given that we now error in `Flake8ImportConventionsOptions`